### DISCUSSION TITLE
Are nested INSERTs possible?

### DISCUSSION THREAD

**Discussion by philip-weber** [supabase/supabase#1578] [Questions] [answered]
Hi there,

I have a one-to-many relation between two tables and would like to insert in both tables with one query. Is this possible with the Supabase-JS Client? Unfortunately I did not find anything regarding this in the docs.

Eg. it would be great if one could insert a new user in the users table and within the same query insert some posts associated to the new user in the posts table. 

For example see the docs from [Prisma](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#create-a-new-user-record-with-two-new-post-records) or [Fauna (with GraphQL)](https://docs.fauna.com/fauna/current/api/graphql/relations#create).

Thank you in advance!

**Comment by steve-chavez** [comment, marked as answer]
Hey @philip-weber,

Is not possible right now. But once https://github.com/PostgREST/postgrest/issues/818 is solved, `supabase-js` will be able to do nested inserts.

**Reply by philip-weber** [reply]
Thank you for the answer. Then I'll keep my fingers crossed that https://github.com/PostgREST/postgrest/issues/818 will be solved sooner or later. Hopefully sooner than later ;)

Regardless, thanks to you and all the other maintainers for your great work on `supabase` and all the libraries it is built on. Keep up the good work!
